### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by abdul-qadir92**
Please Review

### CODE REVIEW

File: src/test/java/Helper/DriverClass.java [hunk 1]
```diff
@@ -13,31 +19,24 @@
...
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
 import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 
 public class DriverClass {
     public WebDriver driver;
     public Properties config;
```
Comment (raveerocks) [inline src/test/java/Helper/DriverClass.java:31]: This variable is not used inside the code.

File: src/test/java/Helper/DriverClass.java [hunk 2]
```diff
@@ -13,31 +19,24 @@
...
     BufferedReader reader;
```
Comment (raveerocks) [inline src/test/java/Helper/DriverClass.java:32]: This variable is not used inside the code

File: src/test/java/Helper/DriverClass.java [hunk 3]
```diff
@@ -13,31 +19,24 @@
...
+    private Local l;
```
Comment (raveerocks) [inline src/test/java/Helper/DriverClass.java:33]: Following a good naming convention often helps

File: src/test/java/Helper/DriverClass.java [hunk 4]
```diff
@@ -13,31 +19,24 @@
...
-        config = new Properties();
-        try {
-            config.load(reader);
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-        String browser = config.getProperty("RequiredBrowser");*/
-        if(browser.contains("Chrome")){
+    public void launch(String config_file, String environment) throws Exception {
+        /*if(browser.contains("Chrome")){
```
Comment (raveerocks) [inline src/test/java/Helper/DriverClass.java:39]: On-premise execution commented. Can this be modified to work with BrowserStack as well?

File: pom.xml [hunk 5]
```diff
@@ -13,22 +13,75 @@
...
+        <suiteXmlFile>config/testng.xml</suiteXmlFile>
+        <suiteXmlFile>config/local.testng.xml</suiteXmlFile>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
     </properties>
-
+    <profiles>
+        <profile>
+            <id>local</id>
+            <build>
```
Comment (raveerocks) [inline pom.xml:54]: Enhancement : Common could could be refactored (Optional)

File: src/test/java/Helper/DriverClass.java (hunk 1 above)
Comment (abdul-qadir92) [reply src/test/java/Helper/DriverClass.java:31]: These imports were used with initial framework which was then modified as per the setup to run on BS and this got redundant.

File: src/test/java/Helper/DriverClass.java (hunk 2 above)
Comment (abdul-qadir92) [reply src/test/java/Helper/DriverClass.java:32]: These imports were used with initial framework which was then modified as per the setup to run on BS and this got redundant.

File: src/test/java/Helper/DriverClass.java (hunk 3 above)
Comment (abdul-qadir92) [reply src/test/java/Helper/DriverClass.java:33]: Noted.

File: src/test/java/Helper/DriverClass.java (hunk 4 above)
Comment (abdul-qadir92) [reply src/test/java/Helper/DriverClass.java:39]: Framework initially fetches the browser type from a config file, but to run on BS it got redundant and JSON config files were introduced to do the parallel test setup. These were commented to highlight the volume of rework needed to set it up for BS.

**Pull request merged**